schemas/user_tasks.proto

```diff
@@ -104,6 +104,7 @@ message UserTaskRun {
 
   // The NodeRun with which the UserTaskRun is associated.
   NodeRunId node_run_id = 11;
+  int64 epoch = 12;
```

Comment:
Please add a comment for this since it is part of the public api.

```
// Current observed epoch of the UserTaskRun, related to the number of times it has been
// updated or re-assigned. Used internally to implement automated reassignment and reminder
// tasks.
```

Reply:
Additionally, the `epoch` can be an `int32` since it will only be re-assigned at most 10-20 times, not 1M+.